typescript/src/classifiers/bedrockClassifier.ts

```diff
@@ -125,19 +126,26 @@ export class BedrockClassifier extends Classifier{
       content: [{ text: inputText }],
     };
 
+    const toolConfig: ToolConfiguration = {
+      tools: this.tools,
+    };
+
+    // ToolChoice is only supported by Anthropic Claude 3 models and by Mistral AI Mistral Large.
+    // https://docs.aws.amazon.com/bedrock/latest/APIReference/API_runtime_ToolChoice.html
+    if (this.modelId.includes("anthropic") || this.modelId.includes("mistral-large")) {
```

Comment:
like above, throw an error.

Reply:
The problem is that some models can use the tool but don't support toolChoice, which about forcing to use the tool.